src/extensions/editable/bootstrap-table-editable.js

```diff
@@ -11,7 +11,7 @@
     onEditableInit () {
       return false
     },
-    onEditableSave (field, row, oldValue, $el) {
+    onEditableSave (field, row, index, oldValue, $el) {
```

Comment:
It's better to use `rowIndex` to be same with documentation.

Reply:
Agree

Reply:
done